From 52db37d1ac3442c87729b09dca42636a19f9efdd Mon Sep 17 00:00:00 2001
From: Sarat Chandra Poluri <sarat.chandra.poluri@intel.com>
Date: Tue, 2 Oct 2018 16:45:16 -0400
Subject: [PATCH 2261/2278] media: intel-ipu4: [VIRT] remove unused functions

Remove unused functions ipu4_virtio_fe_remove() and ipu4_virtio_remove_fe()

Change-Id: I076e4974acfb99a43b09e590cb453c91bd8a40dd
Signed-off-by: Sarat Chandra Poluri <sarat.chandra.poluri@intel.com>
---
 drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.c | 15 ---------------
 drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h |  2 --
 2 files changed, 17 deletions(-)

diff --git a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.c b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.c
index 17b0f06..457c6bd 100644
--- a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.c
+++ b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.c
@@ -59,18 +59,3 @@ struct ipu4_virtio_fe_info *ipu4_virtio_fe_find_by_vmid(int vmid)
 
 	return NULL;
 }
-
-int ipu4_virtio_fe_remove(int client_id)
-{
-	struct ipu4_virtio_fe_info_entry *info_entry;
-	int bkt;
-
-	hash_for_each(ipu4_virtio_fe_hash, bkt, info_entry, node)
-		if (info_entry->info->client_id == client_id) {
-			hash_del(&info_entry->node);
-			kfree(info_entry);
-			return 0;
-		}
-
-	return -ENOENT;
-}
diff --git a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h
index e55685c..8b2260b 100644
--- a/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h
+++ b/drivers/media/pci/intel/virtio/intel-ipu4-virtio-common.h
@@ -124,8 +124,6 @@ void ipu4_virtio_fe_table_init(void);
 
 int ipu4_virtio_fe_add(struct ipu4_virtio_fe_info *fe_info);
 
-int ipu4_virtio_remove_fe(int client_id);
-
 struct ipu4_virtio_fe_info *ipu4_virtio_fe_find(int client_id);
 
 struct ipu4_virtio_fe_info *ipu4_virtio_fe_find_by_vmid(int vmid);
-- 
2.7.4

